chandra_models/compare_models.py

```diff
@@ -0,0 +1,130 @@
+from hashlib import md5
+import json
+from urllib.request import urlopen
+import xija
+
+
+def get_github_model_spec(branch, directory, filename):
```

Comment:
You should be using [get_xija_model_spec](https://sot.github.io/xija/api.html#xija.get_model_spec.get_xija_model_spec) which does the same thing (where the `repo_path` can be the GitHub URL).